pyproject.toml

```
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "generals-bots"
+version = "1.0.0"
+description = "Develop your agent for generals.io!"
+authors = ["Matej Straka <strakammm@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "generals"}]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+numpy = "^2.1.1"
+pettingzoo = "^1.24.3"
+gymnasium = "^0.29.1"
```

lets require `gymnasium` atleast `v1.0.0`

```suggestion
gymnasium = "^0.29.1"
```